lib/login.php

```diff
@@ -147,6 +149,10 @@
     //   but naming this 'user' didn't work at all
     $_SESSION['userdata'] = $auth;
 
+    // You really don't want to store the avatar
+    //   in the SESSION.
+    unset($_SESSION['userdata']['avatar']);
```

Comment:
I don't think that setting `$_SESSION['userdata']` to `$auth` then deleting the bad fields is a good idea.
I believe one should list the important fields that one want to keep, and only copy these few fields.